package.json

```diff
@@ -53,21 +54,21 @@
     "eslint-plugin-promise": "^3.7.0",
     "eslint-plugin-react": "^7.10.0",
     "eslint-plugin-standard": "^3.0.1",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "fibers": "^4.0.2",
```

Comment:
what is this one?

Reply:
A support library for `sass` (the dart version works synchronously) to play nice with node (which does stuff asynchronously). It was recommended by the sass-loader documentation.

Also it turns out we were no longer using `extract-text-webpack-plugin`.